src/cgame/cg_rocket_draw.cpp

```diff
@@ -763,8 +763,39 @@ class CrosshairHudElement : public HudElement {
 
 		if ( crosshair )
 		{
-			CG_GetRocketElementColor( color_ );
-			trap_R_SetColor( color_ );
+			Color::Color crosshairColor;
+			if ( cg_crosshairStyle.Get() == 1 ) {
+				crosshairColor.SetRed( cg_crosshairColorRed.Get() );
+				crosshairColor.SetGreen( cg_crosshairColorGreen.Get() );
+				crosshairColor.SetBlue( cg_crosshairColorBlue.Get() );
+				crosshairColor.SetAlpha( cg_crosshairColorAlpha.Get() );
+			} else {
+				CG_GetRocketElementColor( crosshairColor );
+			}
+			if ( cg_crosshairOutlineStyle.Get() > 0 ) {
+				Color::Color outlineColor(crosshairColor);
+				if ( cg_crosshairOutlineStyle.Get() == 1 ) {
+					float grayScale = outlineColor.Red() * 0.299 + outlineColor.Green() * 0.587 + outlineColor.Blue() * 0.114;
```

Comment:
What are these magic numbers?

Reply:
That is the grayscale formula using luma. I'll add a comment explaining that. I'm not sure if it's completely correct here with regards to linear/sRGB space, but seems to work alright.

Reply:
Ah. Yes, please leave a comment about this and also any deviations from the what is normally textbook so people know what's up.

Reply:
Done